docs/source-pytorch/advanced/model_parallel.rst

```diff
@@ -67,6 +67,39 @@ For training on unreliable mixed GPUs across the internet check out the :doc:`Hi
 ----
 
 
+************************
+Efficient initialization
```

Comment:
Is there a better doc to put this section? Open to suggestions

Reply:
I have this idea:

1. Make an entire standalone page alone just for this (what you described here). 
2. In addition, mention only the relevant part about "configure_model" here in the model parallel docs. 
3. Add relevant info to the mixed precision docs page
4. Add relevant info in the model checkpoint loading docs

What do you think?

Reply:
Do (2), (3), and (4) duplicate part of the information that is in (1)?

If we are thinking of refactoring the overall structure, I would prefer to leave this for a follow-up. I can open an issue about it

Reply:
You directly asked for suggestions, this was my suggestion. 

The feature is useful for a variety of things, putting it in a specific model parallel training doc page is not ideal IMO. Putting it in relevant places or having a separate page is not so far-fetched of an idea, considering the existing documentation structure we have.